renovate.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [":gitSignOff"],
+  "labels": ["release-note-none"],
+  "prConcurrentLimit": 3,
+  "enabledManagers": ["custom.regex", "github-actions"],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": ["^hack/e2e-setup.sh$"],
+      "matchStrings": ["KIND_VERSION=\\${KIND_VERSION:-(?<currentValue>.*?)}"],
+      "datasourceTemplate": "github-releases",
+      "packageNameTemplate": "kubernetes-sigs/kind"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": ["^.github/workflows/integration.yml$"],
+      "matchStrings": ["\\sversion: (?<currentValue>.*?)\\n"],
```

Comment:
Will it consider value as `version: v1.2.3` and also `version: 1.2.3`?

Reply:
How did you test this regex? in `regexr.com` or somewhere else?

Reply:
It would match both, but `v1.2.3` is what's being used.

Yes, or regex101.com.

Reply:
Awesome! :)